### PR TITLE
[react-native-mutli-slider] Don't use children in tests for customMarker

### PR DESCRIPTION
**Pull request by eps1lon**
We plan to remove implicit children from `@types/react`. The following changes are required to pass https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56210. CustomMarker is not actually using children: https://github.com/JackDanielsAndCode/react-native-multi-slider/blob/aec71369ed558812de7546f753494222d12df498/Slider.js#L242-L247

### CODE REVIEW

File: types/react-native-multi-slider/react-native-multi-slider-tests.tsx [hunk 1]
```diff
@@ -59,7 +59,7 @@ class SliderTest extends React.Component {
                         borderRadius: 20,
                         slipDisplacement: 40,
                     }}
-                    customMarker={(props) => <View {...props} />}
+                    customMarker={({ markerStyle, pressed, pressedMarkerStyle, value}) => <View style={pressed ? pressedMarkerStyle : markerStyle}>value: {value}</View>}
```
Comment (Slessi) [inline types/react-native-multi-slider/react-native-multi-slider-tests.tsx:62]: @eps1lon hi! I see you want to fix something relating to `@types/react` deprecating `children`, but since as you say the `CustomMarket` component does not use `children`, why is this change required at all?

Comment (eps1lon) [reply types/react-native-multi-slider/react-native-multi-slider-tests.tsx:62]: Let me check again.

Comment (amcasey) [reply types/react-native-multi-slider/react-native-multi-slider-tests.tsx:62]: @Slessi @eps1lon Where did we land on this?

Comment (eps1lon) [reply types/react-native-multi-slider/react-native-multi-slider-tests.tsx:62]: I forgot about it. Will do tomorrow.

Comment (eps1lon) [reply types/react-native-multi-slider/react-native-multi-slider-tests.tsx:62]: So [`customMarker` is actually used as a component](https://github.com/JackDanielsAndCode/react-native-multi-slider/blob/aec71369ed558812de7546f753494222d12df498/Slider.js#L218) and the [props it receives do not include `children`](https://github.com/JackDanielsAndCode/react-native-multi-slider/blob/aec71369ed558812de7546f753494222d12df498/Slider.js#L256-L261).

The test was previously just spreading props which means TypeScript won't actually complain about excess props:

```tsx
// tsc will not reject this
<View {...props} />;
// tsc will reject this
<View markerStyle={props.markerStyle} />
```

The reason the previous test would be accepte was that `props` had `children` which `View` implements. So there was some intersection between `ViewProps` and `typeof props`. With the removal of implicit `children` that intersection is empty at which point the test would fail.

I hope this makes sense why the previous test was incorrect to begin with and only passed because TypeScript ignores excess props due to spreading.

**Review by Slessi — APPROVED** [review]

**Comment by eps1lon** [comment]
Ready to merge
